### PR TITLE
Adding WoodPlanks and Torches

### PR DESCRIPTION
**Pull request by Petermarcu**
Found an issue while testing round tripping. Adding Torch and WoodPlanks.

### CODE REVIEW

File: Decent.Minecraft.Client/Java/JavaBlock.cs [hunk 1]
```diff
@@ -58,20 +58,21 @@ static JavaBlock()
                 (Bed)new BedFoot((Direction)(d & 0x3), (d & 0x4) != 0) :
                 new BedHead((Direction)(d & 0x3), (d & 0x4) != 0);
             _ctors[Id<Cactus>()] = d => new Cactus(d);
-            _ctors[Id<Chest>()] = d => new Chest(new[] { North, North, South, West, East }[d]);
+            _ctors[Id<Chest>()] = d => new Chest(new[] { Direction.North, Direction.North, Direction.South, Direction.West, Direction.East }[d]);
```
Comment (bleroy) [inline Decent.Minecraft.Client/Java/JavaBlock.cs:61]: Why do you need to fully qualify those?

**Comment by Petermarcu** [comment]
Because Direction and Direction3 both have those same names and are in scope. Thought it was easier to read when it was clear which enumerated was being used.

**Comment by bleroy** [comment]
I'm on the fence as I think the ambiguity is pretty harmless in this case, but ok, fine. Thanks,.

**Comment by Petermarcu** [comment]
It would be less harmful if the values for things of the same name weren't different. I'm personally not a fan of ambiguity to save a few characters. :)

**Comment by bleroy** [comment]
Agreed.

**Pull request merged**